scripts/build/dev-build.ps1

```diff
@@ -62,12 +62,13 @@ try {
     Write-Host "Bin folder to use: $binPath"
     Write-Host "MSBuild: ${msbuildVersion}"
 
+    $buildTarget = "Rebuild"
     if ($restore) {
-        Restore-Packages $msbuildVersion $solutionName
+        $buildTarget = "Restore,Rebuild"
     }
 
     if ($build) {
-        Invoke-MSBuild $msbuildVersion $solutionName `
+        Invoke-MSBuild $msbuildVersion $solutionName /t:"$buildTarget" `
```

Comment:
I'm not sure if we have a scenario where we want to build without a restore. 

We might want to simplify this script and remove the parameter. We could also use `dotnet build` since it does a restore by default if needed.